physics/massive_body_body.hpp

```diff
@@ -156,7 +156,8 @@ inline not_null<std::unique_ptr<MassiveBody>> MassiveBody::ReadFromMessage(
         switch (static_cast<Tag>(enum_value_descriptor->number())) {
           ROTATING_BODY_TAG_VALUE_CASE(ALICE_SUN);
           ROTATING_BODY_TAG_VALUE_CASE(ALICE_WORLD);
-          ROTATING_BODY_TAG_VALUE_CASE(APPARENT_BUBBLE);
+          ROTATING_BODY_TAG_VALUE_CASE(APPARENT);
+          ROTATING_BODY_TAG_VALUE_CASE(APPARENT_WORLD);
```

Comment:
Also add `PILE_UP_PRINCIPAL_AXES`, it's missing in this statement.

Reply:
Done, but we should probably restrict this if not to the frames we actually want to use for bodies, at least to inertial ones which is a requirement of `RotatingBody`…